gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcDirectServerStreamingCallableTest.java

```diff
@@ -160,8 +273,9 @@ public void onError(Throwable t) {
     }
 
     @Override
-    public void onCompleted() {
+    public void onComplete() {
```

Comment:
In the google design of this, we named this method onComplete(), should it be onCompleted() to be consistent with ApiStreamObserver?

Reply:
If we eventually convert all cases of `ApiStreamObserver` to something else, then it won't matter much. I don't have a strong opinion here on the end state between these two.